api/workAllocation2/personService.ts

```diff
@@ -21,10 +22,5 @@ export async function postFindPersonSearch(req: EnhancedRequest, res: Response,
 }
 
 export function getDefaultResult() {
```

Comment:
Can we remove this method if there is nothing we are doing here?

Reply:
Yes, will do actually